Bottom: ce6ed6f8984b691eeae2dd1740b5e403fe88489c
Top:    7e061e92870558c3d6dbb2decabc06eb07698e2f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 00:23:05 +0100

Refresh of draw-c-split-gathering-routine-2

---

diff --git a/src/draw.c b/src/draw.c
index 168884a..03073d4 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -1429,10 +1429,7 @@ void
 DrawElement (ElementTypePtr Element)
 {
   DrawElementPackage (Element);
-  if (Gathering)
-    DrawElementName (Element);
-  else
-    draw_element_name (Element);
+  DrawElementName (Element);
   DrawElementPinsAndPads (Element);
 }
 
@@ -1820,7 +1817,7 @@ static void
 draw_element (ElementTypePtr element)
 {
   DrawElementPackage (element);
-  DrawElementName (element);
+  draw_element_name (Element);
   DrawElementPinsAndPads (element);
 }
